.eslintrc

```diff
@@ -11,33 +11,34 @@
     "Assets": true, // Meteor global
     "Package": true, // Meteor global
     "FS": true, // gridFS,
-    "Alerts": true,  // this needs to get cleaned up, but ignoring for now
+    "Alerts": true, // this needs to get cleaned up, but ignoring for now
     "jest": true,
     "expect": true,
-    "test": true
+    "test": true,
+    "PaystackPop": true
   },
   "parserOptions": {
-     "ecmaVersion": 6,
-     "sourceType": "module",
-      "ecmaFeatures": {
-        "arrowFunctions": true,
-        "blockBindings": true,
-        "classes": true,
-        "defaultParams": true,
-        "destructuring": true,
-        "forOf": true,
-        "generators": false,
-        "modules": true,
-        "objectLiteralComputedProperties": true,
-        "objectLiteralDuplicateProperties": false,
-        "objectLiteralShorthandMethods": true,
-        "objectLiteralShorthandProperties": true,
-        "spread": true,
-        "superInFunctions": true,
-        "templateStrings": true,
-        "jsx": true
-      },
-     "allowImportExportEverywhere": true
+    "ecmaVersion": 6,
```

Comment:
Are all these changes necessary to have for the payment feature to work, I guesse they should not be here then

Reply:
Yes. This change is particularly necessary. I added the PaystackPop and Alerts as a global variable.
Consequently, a restructuring of the indentation